charts/nginx-gateway-fabric/values.yaml

```diff
@@ -122,14 +122,14 @@ nginx:
   image:
     # -- The NGINX image to use.
     repository: ghcr.io/nginxinc/nginx-gateway-fabric/nginx
-    tag: edge
+    tag: 1.5.0
     # @schema
     # enum:
     #   - Always
     #   - IfNotPresent
```

Comment:
Be sure to create an issue for the problem we saw with this field being updated.